### PR TITLE
[FLINK-6496] Port SSl config parameters to ConfigOptions

### PR DESCRIPTION
**Pull request by zentol**
This PR ports all SSL config parameters to config options.

The PR contains 1 commit for each ported parameter.

### CODE REVIEW

File: flink-runtime/src/main/scala/org/apache/flink/runtime/akka/AkkaUtils.scala [hunk 1]
```diff
@@ -26,7 +26,7 @@ import akka.actor._
 import akka.pattern.{ask => akkaAsk}
 import com.typesafe.config.{Config, ConfigFactory}
 import org.apache.flink.api.common.time.Time
-import org.apache.flink.configuration.{AkkaOptions, ConfigConstants, Configuration}
+import org.apache.flink.configuration.{AkkaOptions, ConfigConstants, Configuration, SecurityOptions}
```
Comment (tzulitai) [inline flink-runtime/src/main/scala/org/apache/flink/runtime/akka/AkkaUtils.scala]: `ConfigConstants` can be removed from the imports.

File: flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobClientSslTest.java [hunk 2]
```diff
@@ -35,6 +35,7 @@
 import org.apache.flink.configuration.ConfigConstants;
```
Comment (tzulitai) [inline flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobClientSslTest.java]: This should violate checkstyle. Import non used anymore.

File: flink-runtime/src/test/scala/org/apache/flink/runtime/akka/AkkaSslITCase.scala [hunk 3]
```diff
@@ -20,8 +20,8 @@ package org.apache.flink.runtime.akka
 
 import akka.actor.ActorSystem
 import akka.testkit.{ImplicitSender, TestKit}
-import org.apache.flink.configuration.{AkkaOptions, ConfigConstants, Configuration}
-import org.apache.flink.runtime.testingUtils.{TestingCluster, TestingUtils, ScalaTestingUtils}
+import org.apache.flink.configuration.{AkkaOptions, ConfigConstants, Configuration, SecurityOptions}
```
Comment (tzulitai) [inline flink-runtime/src/test/scala/org/apache/flink/runtime/akka/AkkaSslITCase.scala]: Remove `ConfigConstants`

File: flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobClientSslTest.java (hunk 2 above)
Comment (tzulitai) [inline flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobClientSslTest.java]: Ah wait, checkstyle isn't enabled for this module yet?

Comment (zentol) [inline flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobClientSslTest.java]: no :)

Comment (zentol) [inline flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobClientSslTest.java]: Only small parts of flink-runtime are covered by checkstle.

**Pull request closed without merging**